package.json

```diff
@@ -101,7 +101,7 @@
     "@bbc/psammead-live-label": "2.0.20",
     "@bbc/psammead-locales": "5.0.7",
     "@bbc/psammead-media-indicator": "6.1.1",
-    "@bbc/psammead-most-read": "6.0.31",
+    "@bbc/psammead-most-read": "6.1.0",
```

Comment:
Just observation for another time, these sorts of components that are use case specific may be better being ported into Simorgh as per discussions we've had over the last few months. I think we have a document somewhere discussing this but I can't find it :( 

Just will makes changes like this quicker in future and make psammead more of a general purpose package library.